public/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="icon.png" type="image/x-icon" />
-   
+    <link href='https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css' rel='stylesheet'>
```

Comment:
You can use npm package for the same it will be easier to implement and work with.

Reply:
Thank you for the suggestion, I have made the changes